locale/en_US/manager.xml

```diff
@@ -578,5 +578,6 @@
 	<message key="manager.navigationMenus.logOut.description">A link that will log the user out.</message>
 	<message key="manager.navigationMenus.loggedOut.conditionalWarning">This link will only be displayed when the visitor is logged in.</message>
 	<message key="manager.navigationMenus.loggedIn.conditionalWarning">This link will only be displayed when the visitor is not logged in.</message>
-
+	<message key="manager.navigationMenus.contact.description">Link to a page displaying the Contact information under Context Settings > Contact</message>
```

Comment:
"Context" isn't a word we use with end users. Let's just keep this vague:

"Link to a page displaying your contact information."

Reply:
I changed it. We have some relative issues with other locales for NavigationMenus, I think.

Reply:
You're right. Can you prepare a PR moving those strings into OJS/OMP and using the appropriate terminology? You'll also need to move the strings for all translations.

(I'll cherry-pick this into the stable branch.)